plugins/scripts/upload/trns_transform_Genbank_Genome_to_KBaseGenomeAnnotations_GenomeAnnotation.py

```diff
@@ -1667,118 +1739,123 @@ def upload_genome(shock_service_url=None,
+                                                              "objects":[ { "type":"KBaseGenomeAnnotations.FeatureContainer",
+                                                                            "data":feature_container,
+                                                                            "name": feature_container_object_name,
+                                                                            "hidden":1,
+                                                                            "provenance":feature_container_provenance}]}) 
+            logger.info("Feature Container saved for %s" % (feature_container_object_name)) 
+        else:
+            #Feature container too large
+            #If core type Fail
+            if feature_type in ["gene", "mRNA", "CDS"]:
+                raise Exception("This genome annotation can not be saved due to at least one of the core feature types (gene, CDS, mRNA) being too large for the workspace")
+            #Else do not save the feature container (add waring)
+            else:
+                temp_warning = "Feature type {} will not be made because the resulting object will be too large for the workspace.".format(feature_type)
+                #annotation_quality_warnings.append(temp_warning)
+                sql_cursor.execute("insert into annotation_quality_warnings values(:warning)",(temp_warning,))
+                alias_list = feature_lookup_dict.keys()
+                for alias in alias_list:
+                    f_counter = 0
+#                    alias_tuple_list = feature_lookup_dict[alias]
+                    for temp_fc_ref, temp_fc_id in feature_lookup_dict[alias]:
+                        if feature_type == reverse_feature_container_ref_lookup[temp_fc_ref]:
+                            #HOW TO REMOVE TUPLE FROM LIST IN PLACE?
+                            del( feature_lookup_dict[alias][f_counter])
+                        else:
+                            # important because the positions of all values will be decremented on a delete
+                            # so only increment if you did not delete                   
+                            f_counter += 1
+                    if len(feature_lookup_dict[alias]) == 0:
+                        #Need to remove the alias.
+                        del( feature_lookup_dict[alias]) 
 
-            container_byte_estimate = len(simplejson.dumps(feature_container))  
+    #MAKE THE BAREBONES ANNOTATION QUALITY OBJECT:
+    #LIST OF WARNINGS TO PUT INTO THE AnnotationQualityObject.
+    annotation_quality_warnings = list()
+    annotation_metadata_warnings = list()
 
-            container_feature_lengths = OrderedDict()
-            features_with_sequences_removed_list = list()
-            #1073741824 bytes is a GB.  This gives ~7%cushion for potential unicode 16 characters
-            if container_byte_estimate > 1000000000 : 
```

Comment:
Minor comment: Can we put these constants at the top of the file with some documentation?
e.g.
# Maximum number of bytes allowed in a feature container before we begin to drop features
MAX_FEATURE_CONTAINER_GB = 1

and then include this in the print Exception format.

raise Exception(' ... %.1fGB limit" % (.. , MAX_FEATURE_CONTAINER_GB))